backend/src/utils/route-security.ts

```diff
@@ -50,7 +50,7 @@ const requestSecurityGuardNotebook = async (
   request: OauthFastifyRequest,
   username: string,
 ): Promise<void> => {
-  const user = await getUserName(fastify, request);
+  //const user = await getUserName(fastify, request);
```

Comment:
Can you remove this dead code.

Reply:
Don't remove this 🤔 I think we need it to validate the access of the x-forwarded token. cc @lucferbux

Reply:
This got fixed after a rebase it seems